src/Nethermind/Nethermind.Synchronization/ParallelSync/MultiSyncModeSelector.cs

```diff
@@ -329,32 +337,17 @@ private bool ShouldBeInStateNodesMode(Snapshot best)
         private bool ShouldBeInBeamSyncMode(Snapshot best)
         {
             bool beamSyncEnabled = BeamSyncEnabled;
-            bool fastSyncHasBeenActive = best.Header >= PivotNumber;
-            bool hasAnyPostPivotPeer = AnyPostPivotPeerKnown(best.PeerBlock);
-            bool inStateNodesSync = best.IsInStateSync;
-            bool notInFastSync = !best.IsInFastSync;
-            bool notInAStickyFullSync = !IsInAStickyFullSyncMode(best);
-            bool notHasJustStartedFullSync = !HasJustStartedFullSync(best);
+            bool isInStateSync = best.IsInStateSync;
 
             if (_logger.IsTrace)
             {
                 _logger.Trace("======================== BEAM");
                 _logger.Trace("beamSyncEnabled " + beamSyncEnabled);
-                _logger.Trace("fastSyncHasBeenActive " + beamSyncEnabled);
-                _logger.Trace("hasAnyPostPivotPeer " + beamSyncEnabled);
-                _logger.Trace("inStateNodesSync " + inStateNodesSync);
-                _logger.Trace("notInFastSync " + beamSyncEnabled);
-                _logger.Trace("notInAStickyFullSync " + notInAStickyFullSync);
-                _logger.Trace("notHasJustStartedFullSync " + notHasJustStartedFullSync);
+                _logger.Trace("isInStateSync " + isInStateSync);
```

Comment:
big simplification in this case - we only want to enable beam sync if state sync is enabled